tropestogo/service/crawler/crawler.go

```diff
@@ -149,9 +150,9 @@ func (crawler *ServiceCrawler) CrawlWorkPages(crawlLimit int) (*tropestogo.TvTro
 	return crawledPages, nil
 }
 
-// getNextPageUrl, internal function that looks for the next Work index pagination URL on the current index page
+// getNextPageUri, internal function that looks for the next pagination URI on the current index
```

Comment:
No parece que el cambio de funcionalidad implique el cambio de nombre. En todo caso, el nombre debería tener en cuenta lo que se le pasa como parámetro, incluyendo "for" o "from" o algo así al final.